core/runtime/src/main/java/io/quarkus/runtime/configuration/Address.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.runtime.configuration;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Objects;
+import java.util.Optional;
+
+public final class Address {
```

Comment:
Vert.x 5 introduced Address andAddressResolver - that might be confusing to have something so close in term of naming (and actually locgic too)